flake.nix

```diff
@@ -4,27 +4,23 @@
 
   inputs = {
 
-    CHaP = {
-      url = "github:input-output-hk/cardano-haskell-packages?ref=repo";
-      flake = false;
-    };
-
-    flake-utils.url = "github:numtide/flake-utils";
-
     haskell-nix = {
       url = "github:input-output-hk/haskell.nix";
       inputs.hackage.follows = "hackage";
     };
 
-    # nixpkgs.url = "https://github.com/NixOS/nixpkgs"; 
     nixpkgs.follows = "haskell-nix/nixpkgs-2305";
-    # nixpkgs.follows = "haskell-nix/nixpkgs-unstable";
 
     hackage = {
       url = "github:input-output-hk/hackage.nix";
       flake = false;
     };
 
+    CHaP = {
```

Comment:
Did we decide not to take CHaP and hackage as args?

Reply:
I think we decided that it was irrelevant. Using `follows` is more standard. Also we need CHaP and hackage in iogx's flake in case the user does not override them